app/build.gradle

```diff
@@ -169,6 +169,14 @@ android {
         }
     }
 
+    aaptOptions {
+        // All JavaScript code used internally by GeckoView is packaged in a
+        // file called omni.ja. If this file is compressed in the APK,
+        // GeckoView must uncompress it before it can do anything else which
+        // causes a significant delay on startup.
+        noCompress 'ja'
```

Comment:
Should we document this on AC?

Reply:
Yes, GV and A-C should document this. We should put this in our README of browser-engine-gecko (all 3).

Reply:
👍